deploy/cloudformation/manifest-pipeline.yml

```diff
@@ -515,6 +518,22 @@ Resources:
                       -
                         - !GetAtt ProcessBucket.Arn
                         - "/*"
+        - PolicyName: RBSCS3ImageBucketPermissions
+          PolicyDocument:
+            Version: 2012-10-17
+            Statement:
+              -
+                Action:
+                  - "s3:GetObject"
+                  - "s3:ListBucket"
+                Effect: "Allow"
+                Resource:
+                  - !Ref RBSCS3ImageBucketArn
+                  - Fn::Join:
```

Comment:
I personally prefer sub when it's a simple join like this. Just easier to parse and see that both bucketname and bucketname/* are included. Ex:
```
Resource:
  - !Ref RBSCS3ImageBucketArn
  - !Sub "${RBSCS3ImageBucketArn}/*"
```